root/frontend/src/styles/PreviewCard.scss

```diff
@@ -26,38 +30,43 @@
 	text-overflow: ellipsis;
 	max-width: 150px;
 }
+
 .list {
 	list-style-type: none;
 	margin-top: 0px;
-    :nth-child(n+7) {
-        display: none;
-    }
+	:nth-child(n + 7) {
+		display: none;
+	}
 }
+
 .diamond {
-	height: 10px;
```

Comment:
I think we probably still want to limit the diamond size to something suitable with the text.